inpage.html

```diff
@@ -0,0 +1,433 @@
+        </div>
+        <h4 class="offer-title">English Lessons<br>on-site</h4>
+      </div>
+      <div class="offer-item">
+        <div class="offer-img">
+          <svg class="svg-img" xmlns="http://www.w3.org/2000/svg" width="72" height="72" viewBox="0 0 64 64"><defs><style>.\34 b5e0f8b-7237-4c8b-b762-0790343f8d20,.\35 ba875d0-6564-424d-a79d-a5344763b314{fill:none;stroke:#e8e6e4;stroke-width:2px;}.\35 ba875d0-6564-424d-a79d-a5344763b314{stroke-miterlimit:10;}.\34 b5e0f8b-7237-4c8b-b762-0790343f8d20{stroke-linejoin:round;}</style></defs><title>7</title><g id="bc9375ef-e115-44f5-93ad-f64625e668da" data-name="Layer 2"><g id="35b4094e-87d1-437c-b368-99423516694f" data-name="Isolation Mode"><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="32" y1="39" x2="32" y2="9"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="32" y1="63" x2="32" y2="43"/><circle class="5ba875d0-6564-424d-a79d-a5344763b314" cx="32" cy="5" r="4"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M32,53c2,1,5,2.24,5,5a5,5,0,1,1-9-3"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M32,53c-2-1-7-3-7-7,0-2.46,2-3.81,4-5"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M36,51a5.21,5.21,0,0,0,3-5c0-3.87-5-6-7-7"/><path class="5ba875d0-6564-424d-a79d-a5344763b314" d="M35,25c2,0,5,2,5,6,0,4.42-6,7-8,8-2-1-8-3.58-8-8,0-4,3-6,5-6"/><path class="4b5e0f8b-7237-4c8b-b762-0790343f8d20" d="M29,21H25C11,21,3,16,1,10c14,0,14-5,19-5a4.14,4.14,0,0,1,4,4v3a3,3,0,0,0,3,3h2"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="22" y1="21" x2="22" y2="17"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="18" y1="21" x2="18" y2="16"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="14" y1="20" x2="14" y2="15"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" x1="10" y1="18" x2="10" y2="14"/><path class="4b5e0f8b-7237-4c8b-b762-0790343f8d20" d="M35,21h4c14,0,22-5,24-11C49,10,49,5,44,5a4.14,4.14,0,0,0-4,4v3a3,3,0,0,1-3,3H35"/><line class="5ba875d0-6564-424d-a79d-a5344763b314" 
```

Comment:
Has not been done. Please see 'File changes' of pull request this view didn't put the comment on right line.